src/style.css

```diff
@@ -0,0 +1,124 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir1;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+input {
+  font-family: Avenir1, sans-serif;
+}
+
+body {
+  height: 100vh;
+  font-weight: 300;
+}
+
+.bigBar {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  margin-left: 1.5%;
+  margin-right: 1.5%;
```

Comment:
You can use shorthand property instead
https://developer.mozilla.org/en-US/docs/Web/CSS/margin#syntax